lib/galaxy/managers/users.py

```diff
@@ -133,7 +133,7 @@ def create(self, email=None, username=None, password=None, **kwargs):
         Create a new user.
         """
         self._error_on_duplicate_email(email)
-        user = self.model_class(email=email)
+        user = User(email=email)
```

Comment:
Since that's shared between the toolshed and galaxy we need to keep using self.model_class for now (but I have properly typed it in another PR targeting dev)